### ISSUE TITLE
Not an Issue - How to dynamically create schemas?

### ISSUE THREAD

**Issue by bernardopires** [tuttle/django-schemata#5]
Hello,

it seems like you guys made an awesome app here! From the instructions however, I don't understand how or even if it's possible to do what I'd like to do. I have a SaaS app where schemas would be created dynamically, for example when someone clicks on "trial", it should create a schema for this user and associate a particular subdomain to it. From what I've read, all schemas must be previously defined in a settings file. Is the recommended approach then to dynamically generate this file or is there a smarter way to do this?

Long story short: how do I create schemas on the fly?

Thanks in advance and sorry that I created an issue. I don't see any other way to contact you!

**Comment by tuttle** [comment]
Hey bcarneiro,

thanks for taking a look and expressing your interest.

It's true django-schemata is not very friendly w.r.t. creating schemata on the fly. I chose this approach for our project (which wasn't realized in the end), where it fit well.

But YMMV, I don't think adapting the ideas to on the fly creation wouldn't be too hard.

Unfortunately I can't support the app currently for the reasons mentioned in the README.

Good luck, Vlada
